<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>712</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Insert Beam</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLineEdit" name="led_Length"/>
   </item>
   <item row="10" column="3">
    <widget class="QPushButton" name="addName">
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="lbl_Length">
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Insertion point</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QRadioButton" name="rb_9">
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rb_7">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rb_4">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rb_8">
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="rb_1">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="rb_5">
        <property name="text">
         <string>5 (Origin)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="rb_6">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="rb_2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="rb_3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="18" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="2">
    <widget class="QComboBox" name="cb_Name"/>
   </item>
   <item row="9" column="2">
    <widget class="QComboBox" name="cb_Orientation"/>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Orientation</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>(rotate view to show insertion point)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="1" colspan="2">
    <layout class="QGridLayout" name="gridDim"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Global coords</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridCoords"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
